x-pack/plugins/remote_clusters/public/sections/remote_cluster_list/remote_cluster_table/remote_cluster_table.js

```diff
@@ -180,19 +181,27 @@ export class RemoteClusterTableUi extends Component {
           );
         },
       }, {
-        name: i18n.translate('xpack.remoteClusters.remoteClusterList.table.actionEditAriaLabel', {
-          defaultMessage: 'Edit remote cluster',
-        }),
-        description: i18n.translate('xpack.remoteClusters.remoteClusterList.table.actionEditDescription', {
-          defaultMessage: 'Edit remote cluster',
-        }),
-        color: 'primary',
-        icon: 'pencil',
-        type: 'icon',
-        onClick: () => {},
-        // Implmenet this once EUI supports it.
-        // href: `#${CRUD_APP_BASE_PATH}/edit/${name}`,
-        enabled: ({ isConfiguredByNode }) => !isConfiguredByNode,
+        render: ({ name, isConfiguredByNode }) => {
+          const label = i18n.translate('xpack.remoteClusters.remoteClusterList.table.actionEditDescription', {
+            defaultMessage: 'Edit remote cluster',
+          });
+
+          return (
+            <EuiToolTip
+              content={isConfiguredByNode ? undefined : label}
```

Comment:
a label for why yml configured clusters can't be edited might be nice

Reply:
Good idea!